features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -1149,6 +1156,14 @@
                    timeout="{{cache.idp_cache_by_name.timeout}}"
                    capacity="{{cache.idp_cache_by_name.capacity}}"
                    isDistributed="false"/>
+            {% for cache in cache.manager %}
+             <Cache id="{{cache.id}}"
```

Comment:
No need to have cache.id. We introduced the id value for default configs so we can refer them using the id. For custom cahces we do not need this configuration

Reply:
removed